text/0000-builders.md

```diff
@@ -0,0 +1,101 @@
+- Start Date: 2020-06-25
+- RFC PR: 
+- Svelte Issue: ]
+
+# Sapper Builders
+
+## Summary
+
+This RFC proposes a method of modularising build output from Sapper for ease of deployment across a variety of targets.
+
+## Motivation
+
+Sapper's current build output comprises of a series of files:
+
+1. A build manifest
+1. A client bundle
+1. A server bundle (in non-exported mode)
+1. An (optional) legacy client bundle
+1. Static files
+
+Deploying this is relatively simple, but certainly not optimised for certain deployment strategies such as serverless (lambda), and often relies on third-party, platform-specific builders.
+
+If Sapper were to produce its output in a more modular fashion, it would facilitate the creation of a number of officially supported "builders" for major platforms, as well as opening the doors for third parties to easily produce builders which targetted other platforms.
```

Comment:
One way to think about this is that there should be no `__sapper__/build/index.js`. 

Instead, we should emit enough information for someone to run a completely separate tool against the sapper build output. Something like `sapper serve ...`

This command would read the routes from the given directory and serve them directly. This would also make it straightforward for others to write deploy commands that can package things up for various hosting providers.